master/internal/configpolicy/postgres_task_config_policy.go

```diff
@@ -82,6 +84,9 @@ func GetTaskConfigPolicies(ctx context.Context,
 		Where("workload_type = ?", workloadType).
 		Scan(ctx)
 	if err != nil {
+		if errors.Is(err, db.ErrNotFound) || errors.Cause(err) == sql.ErrNoRows {
+			return &model.TaskConfigPolicies{}, nil
```

Comment:
Does this mean that requests succeed when the config policy DNE?
That means that in the CLI and WebUI, we will have to check for a nil invariant config and constraint, and if both are empty, output something clean to the user.
Can we add some integration test cases for `GetConfigPolicies` for a scope that has no invariant config or constraint?

Reply:
Yes they succeed when the config policy DNE, but I don't think we need to check anything. It already print pretty cleanly:
`{
  "configPolicies": {}
}`

Reply:
added one test case for no policy!

Reply:
Fantastic, thanks!